js/web/lib/wasm/jsep/webgpu/ops/binary-op.ts

```diff
@@ -62,14 +62,24 @@ const createBinaryOpProgramShader =
       let assignment: string;
       if (vectorize) {
         if (doBroadcast) {
-          assignment = `
+          const isAScalar = dimsA.length === 0;
+          const isBScalar = dimsB.length === 0;
```

Comment:
can we extend this from supporting scalar only to supporting 1-sized tensor (eg. dims = [1] , or [1, 1])?